coordinator/src/services/translation/prescription/index.ts

```diff
@@ -166,14 +166,10 @@ function convertDispensingSitePreference(
 function extractText(fhirCommunicationRequests: Array<fhir.CommunicationRequest>): Array<core.Text> {
   return fhirCommunicationRequests
     .flatMap(communicationRequest => communicationRequest.payload)
```

Comment:
As this is a flat map, it will already have extracted the content strings out of the underlying payload object and turned them into `Array<ContentString>`, so line 170 is a little confusing.